src/Dashboard/Grid.php

```diff
@@ -370,16 +371,43 @@ public function show(bool $mini = false, ?string $token = null)
 
         $filters = "";
         if (!$mini) {
-            $filters = <<<HTML
-         <div class='filters_toolbar'>
-            <span class='filters'></span>
-            <span class='filters-control'>
-               <i class="btn btn-sm btn-ghost-secondary fas fa-plus plus-sign add-filter">
-                  <span class='add-filter-lbl'>{$add_filter_lbl}</span>
-               </i>
-            </span>
-         </div>
-HTML;
+            $params = [
+                'change_global_url' => \Config::getFormURL() . '?forcetab=Config$2#section-dashboards',
+                'change_my_url' => \Preference::getSearchURL() . '?forcetab=Config$1#section-dashboards',
+                'filter_toolbar_class' => $placeholder ? 'd-none' : '',
+                'is_placeholder' => $placeholder ? 1 : 0
+            ];
+            $filters = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+                <div class='filters_toolbar m-2 {{ filter_toolbar_class }}'>
```

Comment:
No need to have a specific variable passed to template for this.
```suggestion
                <div class='filters_toolbar m-2 {{ is_placeholder ? "d-none" : "" }}'>
```